### PR TITLE
Update kodi-rbp-git

### PR DESCRIPTION
**Pull request by bh**
It works great on my Raspberry Pi 3. :-)

### CODE REVIEW

File: alarm/kodi-rbp-git/PKGBUILD [hunk 1]
```diff
@@ -24,54 +26,37 @@ makedepends=('hicolor-icon-theme' 'fribidi' 'lzo' 'smbclient' 'libtiff' 'libva'
...
 
 build() {
-  cd "kodi-build"
-
-  # -DCPU=arm1176jzf-s for rpi1
-  # -DCPU=cortex-a7 for rpi2
-  # -DCPU=cortex-a53 for rpi3
-  
-  [[ $CARCH == "armv6h" ]] && _CPU=arm1176jzf-s
-  [[ $CARCH == "armv7h" ]] && _CPU=cortex-a7
```
Comment (invisiblek) [inline alarm/kodi-rbp-git/PKGBUILD]: you should leave this stuff in along with the WITH_CPU on the cmake or you'll break distcc building

File: alarm/kodi-rbp-git/PKGBUILD [hunk 2]
```diff
@@ -24,54 +26,37 @@ makedepends=('hicolor-icon-theme' 'fribidi' 'lzo' 'smbclient' 'libtiff' 'libva'
...
-    -DENABLE_EVENTCLIENTS=ON \
-    -DENABLE_OPENGL=OFF \
-    -DENABLE_X11=OFF \
-    -DENABLE_VAAPI=OFF \
-    -DENABLE_VDPAU=OFF \
-    -DLIRC_DEVICE=/run/lirc/lircd \
-    ../"xbmc-$_tag"/project/cmake
+  cd "$srcdir/kodi-build"
+
+  cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib -DCORE_PLATFORM_NAME=rbpi -DENABLE_EVENTCLIENTS=ON  -DCMAKE_PREFIX_PATH=/opt/vc $srcdir/xbmc
```
Comment (invisiblek) [inline alarm/kodi-rbp-git/PKGBUILD]: maybe leave these lines split out so its easier to see what changed here?

File: alarm/kodi-rbp-git/PKGBUILD [hunk 3]
```diff
@@ -24,54 +26,37 @@ makedepends=('hicolor-icon-theme' 'fribidi' 'lzo' 'smbclient' 'libtiff' 'libva'
...
-source=("xbmc-$_tag.tar.gz::https://github.com/xbmc/xbmc/archive/$_tag.tar.gz"
+             'upower' 'git' 'autoconf' 'java-environment' 'libcrossguid-git' 'dcadec' 'libpulse' 'cwiid'
+             'libfmt-git' 'rapidjson-git'
+)
+source=("$_gitname::git+https://github.com/xbmc/xbmc.git"
         'kodi.service'
         '99-kodi.rules'
-        'https://github.com/popcornmix/xbmc/commit/0c320b6cdd4fb409be45008e6b9042463d54b742.patch'
-        'https://github.com/popcornmix/xbmc/commit/4d983105d7fd65b1d92f2ae2602e6e1cdcaddbe0.patch'
+        'kodi.service'
```
Comment (invisiblek) [inline alarm/kodi-rbp-git/PKGBUILD]: this is now in here twice

**Comment by bh** [comment]
@invisiblek Thanks for your feedback! I'll fix this ;-)

**Comment by flocke** [comment]
@bh Could you upload the three packages somewhere?
I would love to test the Netflix plugin for Kodi 18 (Oreo) and have no way of building the packages myself yet ...

**Comment by bh** [comment]
@flocke I'll make an build for you.

**Comment by bh** [comment]
@invisiblek updated :-)

**Comment by flocke** [comment]
@bh I managed to build it myself using distcc on my Raspberry Pi 3.
Just FYI, if you want to use the Netflix plugin you need to include this patch: https://github.com/popcornmix/xbmc/commit/b4e54bf036c7472e6842d5c960a6fdae36ba19f2

**Pull request closed without merging**